nominations/models.py

```diff
@@ -468,6 +474,7 @@ class Application(models.Model):
         null=True,
     )
 
+    """TODO: remove?"""
```

Comment:
@ericbroder Can we create followup ticket to remove.

Reply:
https://ourrevolution.atlassian.net/browse/TECH-1138